src/tests/ftest/util/command_utils.py

```diff
@@ -20,6 +20,7 @@
 from command_utils_base import \
     BasicParameter, CommandWithParameters, \
     EnvironmentVariables, LogParameter
+from daos.src.tests.ftest.util.command_utils_base import ObjectWithParameters
```

Comment:
(pylint-import-error) Unable to import 'daos.src.tests.ftest.util.command_utils_base'